### ISSUE TITLE
Deprecate/remove default tags.

### ISSUE THREAD

**Issue by back2dos** [haxetink/tink_hxx#52]
Default tags are used to bring `<div />` and such into scope. They complicate tag resolution *a lot* (which in turns makes supporting IDE services all the harder) and they really just save the user one `*` import (which can go into `import.hx`).

I'm inclined to remove them straight away, in favor of a clear error message - unfortunately, the user will get to see it in builds, but not in diagnostics, but I suppose most hxx (i.e. coconut) users are pretty used to IDE services being weird.

**Comment by kLabz** [comment]
Will that impact react-next? If so, any idea what I should be doing to be compatible?

**Comment by back2dos** [comment]
Nope, react-next uses its own generator, which simply put *knows* what a `div` is. It will be unaffected.

**Comment by back2dos** [comment]
Well, after removing them and trying to migrate a bigger project I've realized this is a pita. Should deprecate instead.
